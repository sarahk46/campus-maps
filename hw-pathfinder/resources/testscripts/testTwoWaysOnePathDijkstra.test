# Find a path with Dijkstra's algorithm when
# only one path exists for the two nodes, but it goes both ways.
# Checks for this two-way property by changing the ordering of arguments as well.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B

AddEdge simpleGraph A B 10.0
AddEdge simpleGraph B A 10.0

FindPath simpleGraph A B
FindPath simpleGraph B A