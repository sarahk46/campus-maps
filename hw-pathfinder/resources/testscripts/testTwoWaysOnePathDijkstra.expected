# Find a path with Dijkstra's algorithm when
# only one path exists for the two nodes, but it goes both ways.
# Checks for this two-way property by changing the ordering of arguments as well.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph

added edge 10.000 from A to B in simpleGraph
added edge 10.000 from B to A in simpleGraph

path from A to B:
A to B with weight 10.000
total cost: 10.000
path from B to A:
B to A with weight 10.000
total cost: 10.000